packages/shared/src/hooks/post/usePostBoostEstimation.ts

```diff
@@ -67,6 +65,10 @@ export const usePostBoostEstimation = ({
           cache.state.fetchFailureCount,
         );
 
+        if (retries === 0 && !canBoost) {
+          return 1; // initial fetch
+        }
```

Comment:
do you need this, it should initial fetch either way, this is just after how much to retry?